src/gui/torrentcontentmodel.cpp

```diff
@@ -113,31 +106,23 @@ namespace
         {
             const QString ext = info.suffix();
             if (!ext.isEmpty()) {
-                const QPixmap pixmap = pixmapForExtension(ext, QSize(32, 32));
-                if (!pixmap.isNull())
-                    return QIcon(pixmap);
+                auto cacheIter = m_iconCache.find(ext);
```

Comment:
It should come in a separate commit.
Or since it is out-of-scope for this PR, you can leave it unimplemented.